app/components/forms/wizard/sponsors-step.js

```diff
@@ -70,6 +74,12 @@ export default Component.extend(FormMixin, {
       });
     }
   },
+    unpublish() {
+      this.onValid(() => {
+        this.set('data.event.state', 'draft');
+        this.sendAction('save', this.data);
+      });
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6112249)